module/x/gravity/keeper/attestation_handler.go

```diff
@@ -21,317 +22,510 @@ var _ types.DistributionKeeper = (*distrkeeper.Keeper)(nil)
 // AttestationHandler processes `observed` Attestations
 type AttestationHandler struct {
 	// NOTE: If you add anything to this struct, add a nil check to ValidateMembers below!
-	keeper     *Keeper
-	bankKeeper *bankkeeper.BaseKeeper
-	distKeeper *distrkeeper.Keeper
```

Comment:
this is a really great cleanup, I shudder to think about what would happen if these duplicated pointers ended up being to different things.